components/camera/transformpipeline/mods.go

```diff
@@ -187,16 +188,44 @@ func newCropTransform(
 	if conf.YMin >= conf.YMax {
 		return nil, camera.UnspecifiedStream, errors.New("cannot crop image to 0 height (y_min is >= y_max)")
 	}
-	cropRect := image.Rect(conf.XMin, conf.YMin, conf.XMax, conf.YMax)
+	cropRect := image.Rectangle{}
+	cropRel := []float64{}
+	if conf.XMax < 1. || conf.YMax < 1. {
+		if conf.XMin <= 1. && conf.YMin <= 1. && conf.XMax <= 1. && conf.YMax <= 1. {
```

Comment:
Mild preference for using `1.0` instead of just `1.`, to emphasize that they're floats. It's easy for someone in a hurry to overlook the dot by itself. but I don't feel strongly.